plugin/trino-hive/src/main/java/io/trino/plugin/hive/security/SqlStandardAccessControl.java

```diff
@@ -491,7 +491,7 @@ public void checkCanShowRoleAuthorizationDescriptors(ConnectorSecurityContext co
     public void checkCanShowRoles(ConnectorSecurityContext context)
     {
         if (!isAdmin(context)) {
-            denyShowRoles();
+            denyShowRoles(Optional.of(catalogName));
```

Comment:
I like the change, since it improves (restores) nice error reporting when user cannot SHOW ROLES in hive. 
i hope this will require test updates in product test and should go into separate commit.